coral/src/app/pages/not-found/index.tsx

```diff
@@ -8,13 +8,13 @@ const NotFound = () => {
           Page not found
         </Typography.Heading>
 
-        <Typography.LargeText>
+        <Typography.DefaultStrong>
           Sorry, the page you are looking for does not exist.
-        </Typography.LargeText>
+        </Typography.DefaultStrong>
 
-        <Typography.MediumText>
+        <Typography.Default>
```

Comment:
Same for here, the substitution suggested is `DefaultStrong`

![Screenshot 2023-10-09 at 10 21 28](https://github.com/Aiven-Open/klaw/assets/20607294/2cac72ee-a08d-4b33-a22b-9be1aba1fcf7)